org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this?

Reply:
by accident created by project import action, will delete this

Reply:
Looks like it is still here